src/main/steps/applicant/confirm-contact-details/checkanswers/ApplicantConfirmContactDetailsGetController.ts

```diff
@@ -0,0 +1,122 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { getCaseApi } from '../../../../app/case/CaseApi';
+import { CONFIDENTIAL_DETAILS } from '../../../../app/case/definition';
+import { AppRequest } from '../../../../app/controller/AppRequest';
+import { GetController } from '../../../../app/controller/GetController';
+@autobind
+export default class ApplicantConfirmContactDetailsGetController extends GetController {
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    const redirect = false;
+
+    if (req.session?.user) {
+      res.locals.isLoggedIn = true;
+      req.locals.api = getCaseApi(req.session.user, req.locals.logger);
+      if (!req.session.userCase) {
```

Comment:
Please remove this commented code and the 'if' condition, as we have commented everything inside if loop

Reply:
agreed, removed unwanted commented code